services/rds/src/main/java/org/finra/gatekeeper/services/auth/GatekeeperRoleService.java

```diff
@@ -164,8 +170,46 @@ public GatekeeperRdsRole getRole(){
         return checkGatekeeperRdsRole();
     }
 
+    public Map<String, RoleMembership> getRoleMemberships() {
+        Map<String, RoleMembership> roleMemberships = new HashMap<>();
+        Set<String> dbaMemberships = getDbaMemberships();
+        Set<String> opsMemberships = getOpsMemberships();
+        Map<String, Set<String>> devMemberships = getDevMemberships();
+
+        for(String application : dbaMemberships) {
```

Comment:
You could clean this up by doing this
```suggestion
dbaMemberships.forEach(membership -> {
    roleMemberships.get(application).put(GatekeeperRdsRole.DBA, new HashSet<String>(Arrays.asList("DEV","QA","PROD");
}
```